File: 프로그래머스/lv0/120907. OX퀴즈/OX퀴즈.py
def solution(quiz):
    def rightCal(el):
        [x, d, y, e, z] = el;
        if d == '+':
            return "O" if int(x) + int(y) == int(z) else "X"
        else:
            return "O" if int(x) - int(y) == int(z) else "X"
        
    return list(map(lambda el: rightCal(el), [i.split(' ') for i in quiz]))

File: 프로그래머스/lv0/120907. OX퀴즈/OX퀴즈.js
function solution(quiz) {
    return quiz.map(el => el.split(' '))
        .map(([x, d, y, _, z]) => d === '+' && Number(x) + Number(y) === Number(z)           ? "O" : d === '-' && Number(x) - Number(y) === Number(z) ? "O" : "X")
}